Tests/SwiftCrossUITests/SwiftCrossUIInternalsTests.swift

```diff
@@ -0,0 +1,180 @@
+/// Required Imports
+import Foundation
+import Testing
+
+@testable import SwiftCrossUI
+
+@Suite(
+    "SwiftCrossUI's Internal Types",
+    .tags(.Internal)
+)
+struct InternalTests {
```

Comment:
These aren't internal, they are testing parts of SwiftCrossUI's public interface